pkg/policy/api/ingress.go

```diff
@@ -72,6 +74,19 @@ type IngressCommonRule struct {
 	// +kubebuilder:validation:Optional
 	FromEntities EntitySlice `json:"fromEntities,omitempty"`
 
+	// FromGroups is a directive that allows the integration with multiple outside
+	// providers. Currently, only AWS is supported, and the rule can select by
+	// multiple sub directives:
+	//
+	// Example:
+	// FromGroups:
+	// - aws:
+	//     securityGroupsIds:
+	//     - 'sg-XXXXXXXXXXXXX'
+	//
+	// +kubebuilder:validation:Optional
+	FromGroups []Groups `json:"FromGroups,omitempty"`
```

Comment:
```suggestion
	FromGroups []Groups `json:"fromGroups,omitempty"`
```

to conform with the other fields.